core/templates/dev/head/pages/exploration-editor-page/translation-tab/audio-translation-bar/audio-translation-bar.directive.ts

```diff
@@ -289,14 +289,16 @@ angular.module('oppia').directive('audioTranslationBar', [
             $scope.showRecorderWarning = false;
             AssetsBackendApiService.saveAudio(
               ContextService.getExplorationId(), filename,
-              recordedAudioFile).then(function() {
+              recordedAudioFile).then(function(response) {
               if ($scope.audioIsUpdating) {
                 StateRecordedVoiceoversService.displayed.deleteVoiceover(
                   contentId, languageCode);
                 $scope.audioIsUpdating = false;
               }
               StateRecordedVoiceoversService.displayed.addVoiceover(
-                contentId, languageCode, filename, recordedAudioFile.size);
+                contentId, languageCode, filename, recordedAudioFile.size,
+                response.duration_secs);
+              $scope.duration_secs = Math.floor(response.duration_secs);
```

Comment:
Here and elsewhere, use camelCase in JS: $scope.durationSecs

The only exception is dicts from the backend but we should keep that minimal. Typically this is done by converting to a domain object as soon as possible, but I understand that this isn't done yet for some of the HTTP calls.

Reply:
done.